README.md

```diff
@@ -1,9 +1,11 @@
 
-Copyright 2023 Monaco F. J. <monaco@usp.br> 
-Copyright 2024 The Authors of Coral
-
```

Comment:
Sorry for the nitpick, but this should be kept too, right?

Reply:
It's ok. But I don't think so... Isn't that redundant to the content described in the SPDX format?

Reply:
![image](https://github.com/user-attachments/assets/f3e24c79-6567-443e-a035-270698cc8c39)
The SPDX is a comment, so it can't be seen by users just visiting the page. The way I understand it, this removed bit is just there for the sake of transparency and communicating to the user the license of the entire project (and not just the metadata of the README). But you can choose if you want to keep it or not

Reply:
Makes sense. Updated README.